info-daemn/Makefile

```diff
@@ -0,0 +1,35 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=info-daemon
+PKG_VERSION:=0.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/CreatorDev/ci40-info-daemon.git
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
```

Comment:
I updated the tag https://github.com/CreatorDev/ci40-info-daemon/releases/tag/0.1.0